src/components/ChatLog.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+
+
+const ChatLog = (props) => {
+    const chatComponentsArray = props.entries.map((entry) => {
+        return (
+            <ChatEntry 
+            key={entry.id} 
+            id={entry.id}
+            body={entry.body}
+            sender={entry.sender}
+            timeStamp = {entry.timeStamp}
+            liked = {entry.liked}
+            toggleLike = {props.toggleLike}
+            />
+    );
+    });
+    return (
+        <div>{chatComponentsArray}</div>) 
+};
+
+ChatLog.propTypes = {
+    entry: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
Since we know the shape of the object that is in the array, we should explicitly validate the shape of the object here for an added layer of validation to prevent your protect your program from errors. 

Also note that you have .isRequired on arrayOf() but you'll also. need to have .isRequired on the elements inside arrayOf(). 


It might also make sense to rename `entry` to `entries` since the value is an array of many entries

```javascript
ChatLog.propTypes = {
  entry: PropTypes.arrayOf(PropTypes.shape({
    id: PropTypes.number.isRequired,
    sender: PropTypes.string.isRequired,
    body: PropTypes.string.isRequired,
    timeStamp: PropTypes.string.isRequired,
    liked: PropTypes.bool
  })).isRequired,
  toggleLike: PropTypes.func.isRequired
};
```